ring-jetty-adapter/src/ring/adapter/jetty.clj

```diff
@@ -266,6 +282,7 @@
   :async-timeout-handler  - an async handler to handle an async context timeout
   :port                   - the port to listen on (defaults to 80)
   :host                   - the hostname to listen on
+  :unix-socket            - the unix domain docket path to listen on
```

Comment:
"unix domain docket" should probably be "unix domain socket".

Reply:
Fixed as requested!